features/fix.feature

```diff
@@ -203,7 +203,82 @@ Feature: Ua fix command behaviour
         """
```

Comment:
Not for this PR, but I'm noticing that this scenario is hundreds of lines long. We should break it up in the future

Reply:
Yep, we also need to reevaluate if we need all of those fix scenarios, because maybe we don't